### DISCUSSION TITLE
Where did `D3D9ON12_ARGS` go?

### DISCUSSION THREAD

**Discussion by AArnott** [microsoft/win32metadata#735] [Q&A] [answered]
The 15.0.1 release removed `D3D9ON12_ARGS`, which was present as late as the 14.0.3 release.
I can't find mention of this in the release notes.

Missing this type is breaking one of my projection's tests which generated that type specifically because it had an inline array of `IUnknown` objects, which required special handling.

Where did this type move to?

**Comment by sotteson1** [comment, marked as answer]
Sorry about that. I see it got lost when I was refactoring the D3D stuff. I'll bring it back with a new release today.
